topology/src/main/java/io/camunda/zeebe/topology/state/ClusterTopology.java

```diff
@@ -33,19 +33,19 @@
 public record ClusterTopology(
     long version, Map<MemberId, MemberState> members, ClusterChangePlan changes) {
 
-  static ClusterTopology init() {
+  public static ClusterTopology init() {
     return new ClusterTopology(0, Map.of(), ClusterChangePlan.empty());
   }
 
-  static ClusterTopology decode(final byte[] serializedTopology) throws IOException {
+  public static ClusterTopology decode(final byte[] serializedTopology) throws IOException {
```

Comment:
🔧️ We could remove the JSON serialization and serialize it as a protobuf message

Reply:
That can be done in a followup.